src/pages/home.tsx

```diff
@@ -57,43 +30,30 @@ const Home = () => {
 
   return (
     // used to possibly center the content
-    <div className="relative mx-auto flex h-full w-full max-w-4xl flex-row justify-center overflow-y-scroll pt-8 pb-10 @container">
-      {/* main content container */}
-      <div className="flex h-fit min-h-full w-full grow flex-col items-start gap-2">
-        <h1 className="font-title text-2xlarge font-bold tracking-tight text-gray-300">
-          {`Welcome, ${
-            token && !isLoading && !isError && user
-              ? `${user.username}`
-              : 'Guest'
-          }!`}
-        </h1>
-        <h3 className="mb-4 text-h3 font-medium italic tracking-medium text-white/50">
-          {' '}
-          Select or create an instance to continue.
-        </h3>
-        <DashboardCard>
-          <div className="my-8 grid grid-cols-2 gap-10">
-            <div>
-              <PerformanceGraph
-                title="CPU load"
-                color="#62DD76"
-                backgroundColor="#61AE3240"
-                getter={getCpuUsage}
-                unit="%"
-              />
-            </div>
-            <div>
-              <PerformanceGraph
-                title="Memory load"
-                color="#62DD76"
-                backgroundColor="#61AE3240"
-                getter={getRamUsage}
-                unit="GiB"
-              />
-            </div>
-          </div>
-        </DashboardCard>
+    <div className="flex flex-col items-center justify-center text-gray-faded/30">
+      <div className="flex h-28 w-28 items-center justify-center rounded-full border border-dashed border-gray-faded/30 align-middle">
+        <img
+          src="/assets/placeholder-cube.png"
```

Comment:
try to add the css `{imageRendering: 'pixelated'}` to disable image upscale interpolation